docs/Advanced-Usage/FireAxe-Partitioning-onto-Multiple-FPGAs/FireAxe-Overview.rst

```diff
@@ -0,0 +1,73 @@
+FireAxe Overview
+=============================================
+
+Since the initial release of FireSim, the proliferation of open hardware IP has
+enabled researchers to generate new SoC configurations that no longer fit in a
+single FPGA. FireAxe builds on top of FireSim to enable *partitioning of large
+designs onto multiple FPGAs*, overcoming the single-FPGA limitation for monolithic RTL designs.
+
+Partition Modes
+==================
+FireSim uses host-decoupling to control the advancement of the simulator, and
```

Comment:
Why is this sentence necessary, I would just get into the modes and not mention that there are "weird" things (you can leave that to the paper)